extension/content.js

```diff
@@ -294,6 +307,10 @@ document.addEventListener('DOMContentLoaded', () => {
 				addReadmeEditButton();
 			}
 
+			if (pageDetect.isPRList() || pageDetect.isIssueList()) {
+				showRecentPushedBranches();
```

Comment:
Nitpick: Could you rename `showRecentPushedBranches` to `showRecentlyPushedBranches`? Like in the PR title?